src/assets/src/components/queueManager.tsx

```diff
@@ -168,6 +168,14 @@ function QueueManager(props: QueueManagerProps) {
                 <Col md={2}><div id='created'>Created</div></Col>
                 <Col md={6}><div aria-labelledby='created'><DateDisplay date={props.queue.created_at} /></div></Col>
             </Row>
+            <Row noGutters className={spacingClass}>
+                <Col md={2}><div id='meetingLocation'>Meeting Location</div></Col>
+                <Col md={6}><div aria-labelledby='meetingLocation'>{props.queue.meeting_location}</div></Col>
+            </Row>
```

Comment:
Nice idea, I thought about this, we may want to add something like this later on.